types/chai/chai-tests.ts

```diff
@@ -2,7 +2,7 @@
 import * as chai from 'chai';
 
 const expect = chai.expect;
-const assert = chai.assert;
+const assert: typeof chai.assert = chai.assert; // assertion aliases require explicit type annotation
```

Comment:
This change doesn't appear to do anything. With or without the additional annotation, the type of `assert` is still `Chai.AssertStatic`

Reply:
Nevermind, I understand this one now.